### DISCUSSION TITLE
setLayerType LAYER_TYPE_SOFTWARE

### DISCUSSION THREAD

**Discussion by nickfaces** [KevinnZou/compose-webview-multiplatform#69] [General]
Hi! Do I understand correctly that the Software Layer is now always used?
And what is the impact on the performance of this, as I observe that scrolling in webview on Android (on iOS - ok) works slowly (especially noticeable on horizontal scrolling), and sometimes it works somehow every other time. Could it be the SoftwareLayer that is causing this?

**Comment by KevinnZou** [comment]
@nickfaces Thanks for your feedback! In version 1.7.8, I have set the Software Layer as the default solution to address the [issue](https://github.com/KevinnZou/compose-webview-multiplatform/issues/64). If scrolling worked well in the previous version, then the problem lies with the Software Layer. In the next version, I will make it optional. I apologize for any inconvenience caused.

**Reply by nickfaces** [reply]
@KevinnZou I worked with 1.8.0-SNAPHOT version (because jsBridge is needed), but as far as I understand, changes of Layer have also been made in this branch.

**Reply by KevinnZou** [reply]
Yes, it is also included in this branch. However, I will switch to Hardware Layer and release the 1.8.0 today. Sorry for the inconvenience caused.
